python/Ganga/GPIDev/Lib/Job/Job.py

```diff
@@ -50,22 +50,28 @@ def lazyLoadJobApplication(this_job):
     return lazyLoadJobObject(this_job, 'application')
 
 
-def lazyLoadJobObject(raw_job, this_attr):
+def lazyLoadJobObject(raw_job, this_attr, dont_eval=False):
```

Comment:
`dont_eval=False` is a potentially confusing double negative. Perhaps `do_eval=True` is better?